tools/release/publish_release.sh

```diff
@@ -404,6 +412,17 @@ tag() {
 
 publish() {
     if [ "$dry_run" = "false" ]; then
+        if [[ "$main_release" == "true" ]]; then
+            article_url="https://fleetdm.com/releases/fleet-$target_milestone"
+            article_published=`curl -is "$article_url" | head -n 1 | awk '{print $2}'`
+            if [[ "$article_published" != "200" ]]; then
+                echo "Coulndn't find article at '$article_url'"
+                exit 1
+            fi
+
+            # TODO Publish Linkedin post about release article here and save url
+            linkedin_post_url=""
```

Comment:
For this, we need to publish the LinkedIn post manually, then populate this string before running the release script? That may not work, because we can't post to LinkedIn until the release is published. Maybe I'm misunderstanding the flow?